source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -88,32 +88,32 @@ def generate_coord_id(self, obj):
         return None
 
 
+def make_admin(admin):
+    res = {
+        'id': admin['id'],
+        'insee': admin['insee'],
+        'name': admin['name'],
+        'label': admin['label'],
+        'level': admin['level'],
+        'coord': {'lon': str(admin['coord']['lon']), 'lat': str(admin['coord']['lat'])},
+    }
+    zip_codes = admin.get('zip_codes', [])
+    if all(zip_code == "" for zip_code in zip_codes):
+        pass
+    elif len(zip_codes) == 1:
+        res['zip_code'] = zip_codes[0]
+    else:
+        res['zip_code'] = '{}-{}'.format(min(zip_codes), max(zip_codes))
+    return res
```

Comment:
We can use https://github.com/hove-io/navitia/blob/0ed9608f23492eac15a9826ad01904ed46d13ccd/source/jormungandr/jormungandr/autocomplete/geocodejson.py#L72 instead.

Reply:
Done